simplecpp.cpp

```diff
@@ -1724,11 +1724,31 @@ namespace simplecpp {
      */
     std::string simplifyPath(std::string path)
     {
+        const bool isUnc = path.compare(0,2,"//") == 0;
+
+        // Remove ./, .//, ./// etc. at the beginning
+        while (path.compare(0,2,"./") == 0) { // remove "./././"
```

Comment:
this should be after `// replace backslash separators` so .\ is handled the same way